specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/examples/migrateFromSmartDetectionStartMigration.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "70afbb35-0463-4758-badf-d5aab5bce520",
+    "api-version": "2021-01-01-preview",
+    "parameters": {
```

Comment:
>"parameters" [](start = 4, length = 12)

I believe this should be "properties" not parameters.

Reply:
looking at other example files in the repo, there is paramters, and then another parameters, and only then properties as the 3rd level.
also the doc seem find with what i've done.
https://review.docs.microsoft.com/en-us/rest/api/documentation-preview/migratefromsmartdetection/startmigration?view=azure-rest-preview&branch=result_openapiHub_production_e62f880f06b4
are you sure?

Reply:
Are you sure? looking in other examples in the repo, there are 2 levels of parameters and then properties comes as 3rd level (which I dont need).
It also looks fine in the doc: https://review.docs.microsoft.com/en-us/rest/api/documentation-preview/migratefromsmartdetection/startmigration?view=azure-rest-preview&branch=result_openapiHub_production_e62f880f06b4

Reply:
I see, it works because you have "parameters" as name of your POST body:

```
"name": "parameters",
```

Which is a little bit odd, because usually the property holding the request body has a name related to the operation, something like "migration" in your case or something like that.

---
In reply to: [570234800](https://github.com/Azure/azure-rest-api-specs/pull/12748#discussion_r570234800) [](ancestors = 570234800)